getting_started/first_3d_game/01.game_setup.rst

```diff
@@ -0,0 +1,164 @@
+.. _doc_first_3d_game_game_area:
+
+Setting up the game area
+========================
+
+In this first part, we're going to set up the game area. Let's get started by
+importing the start assets and setting up the game scene.
+
+We've prepared a Godot project with the 3D models and sounds we'll use for this
+tutorial, linked in the index page. If you haven't done so yet, you can download
+the archive here: `Squash the Creeps assets
+<https://github.com/GDQuest/godot-3d-dodge-the-creeps/releases/tag/1.0.0>`__.
+
+Once you downloaded it, extract the .zip archive on your computer. Open the
+Godot project manager and click the *Import* button.
+
+|image1|
+
+In the import popup, enter the full path to the freshly created directory
+``squash_the_creeps_start/``. You can click the *Browse* button on the right to
+open a file browser and navigate to the ``project.godot`` file the folder
+contains.
+
+|image2|
+
+Click *Import & Edit* to open the project in the editor.
+
+|image3|
+
+The start project contains an icon and two folders: ``art/`` and ``fonts/``.
+There, you will find the art assets and music we'll use in the game.
+
+|image4|
+
+There are two 3D models, ``player.glb`` and ``mob.glb``, some materials that
```

Comment:
It's maybe a little pedantic, but should probably wrap ``player.glb`` and ``mob.glb`` in parenthesis instead of including them in the list. So:
> There are two 3D models (``player.glb`` and ``mob.glb``), some materials that
> belong to these models, and a music track.

Reply:
I think it's better as-is actually.

Reply:
Agreed with aaronfranke, note too that the guideline is to avoid parentheses mid-sentence, except in some cases like providing the MacOS-version of a keyboard shortcut.